src/huntsman/pocs/dome/musca.py

```diff
@@ -269,8 +269,10 @@ def _get_shutter_status_dict(self):
         self._write_musca(Protocol.GET_STATUS)
         shutter_status_dict = {}
         num_lines = len(Protocol.VALID_DEVICE)
-        for i in range(num_lines + 1):
+        for i in range(num_lines + 1):  # Add one for the beginning 'Status' key
             k, v = self.serial.read().strip().split(':')
+            if k == 'Status':
+                continue
             if k == Protocol.SOLAR_ARRAY or k == Protocol.BATTERY:
                 v = float(v)
             shutter_status_dict[k] = v
```

Comment:
Let's actually switch the logic a bit, which will also handle the coverage better:

```suggestion
            if k == Protocol.SOLAR_ARRAY or k == Protocol.BATTERY:
                v = float(v)
            if k in Protocol.VALID_DEVICE:
                shutter_status_dict[k] = v
```